src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -3,15 +3,18 @@
 Release History
 ===============
 
-* `deployment create`: Fix issue where type field was case-sensitive.
-
-2.1.10
+2.1.11
 ++++++
+* `deployment create`: Fix issue where type field was case-sensitive.
```

Comment:
This looks wrong. The new line should be 2.1.11 but the following 4 lines should be part of 2.1.10...

Reply:
@tjprescott this is from the previous release. Looks good when I merged locally and compared to pypi.